README.md

```diff
@@ -209,6 +209,11 @@ Table of Contents
   * https://postmarkapp.com - First 25,000 emails are free
   * https://www.zoho.com/mail/ - Free Email management and collaboration for up to 10 users.
   * http://moosend.com/ — Mailing list management service. Free account for 6 months for startups.
+  * https://sendsay.ru/ — 200 subscribers and 1,000 emails per month for free
```

Comment:
might be missing it but cant find a english page for this, its probably useful but way too niche to be a russian language only one